src/data/finset/lattice.lean

```diff
@@ -396,7 +395,7 @@ section distrib_lattice
 variables [distrib_lattice α]
 
 section order_bot
-variables [order_bot α] {s : finset β} {f : β → α} {a : α}
+variables [order_bot α] {f : ι → α} {g : κ → α} {s : finset ι} {t : finset κ} {a : α}
```

Comment:
Any reason you swapped the order of `s` and `f`? The old order matches the argument order to `sup`, right?